src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,106 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.EntityManager;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinTable;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.validation.constraints.NotNull;
+
+
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+public class User {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private long id;
+
+	@NotNull
+	private String userName;
+	
+	@NotNull
+	private String name;
+	
+	@NotNull
+	private LocalDate birthDate;
+	
+	@NotNull
+	@ManyToMany(cascade = CascadeType.ALL)
```

Comment:
En realidad no iría ALL, sino MERGE y REFRESH. Esto igual pienso explicarlo en la próxima call para dar un poco de luz acerca del tema :) 
Pero básicamente es así: 

Al decir ALL, estamos diciendo: Persist, Merge, Refresh, Delete y Detach.
Acá dejo una breve explicación de cada tipo:

Detach is only so that something gets detached along the cascade when you detach an object from the persistence context manually. This almost never happens, so we won't focus on this point.

Persist says: If you persist the Users instance to the DB, then persist every item that this collection has in the DB as well.
Delete says: If you delete the Users instance from the DB, then delete every item that this collection has in the DB as well.
Merge says: If you changed the in-memory values of the collection, then merge this changes to the DB so these are reflected there.
Refresh says: If something changed in the database values of the collection, then refresh those changes in the in-memory values.